src/AutoUI/Lenses/LensSelection.tsx

```diff
@@ -21,26 +23,25 @@ export const LensSelection = ({
 	}
 	return (
 		<Box {...rest}>
-			<ButtonGroup>
+			{/* Using `selected` on each item instead of `value` on the group as the Tooltip break the button group logic */}
+			<ToggleButtonGroup exclusive aria-label="Lenses">
```

Comment:
Do we need to be using translations later for `aria-label`? I don't recall but I believe this is a property that helps with accessibility, right?